### ISSUE TITLE
New tiles for Hero

### ISSUE THREAD

**Issue by psywombats** [psywombats/mgne#135]
Need a new tileset for some stuff in Hero. Aside from different floors/walls/whatever optional stuff, some things we'll definitely need:

* Gate entrance (to the ~demon world~) usable on overworld as well
* Ruins upper chip (or wall tile?), so that Hero doesn't look exactly like a world 1 town
* Castle ramparts (not like an RM square of nine sort of thing, but at least a horizontal rampart chip so that you can go on the roof of a building, in a separate map)
* Castle overworld chip

Basically I think Hero should be one big town without an "overworld." The building that the player starts in is the only safe area and monsters will attack in the city proper. Would be interested if you have a different approach though. Goals for Hero are to just get a couple more strategic encounters, upgrade the player's equipment and cashtstack, and get them prepared for a long haul through the hub world.

**Comment by crumley467** [comment]
Sounds good to me. I have a few things for hero kicking around already with one or two ideas for more in addition to what you mention here

**Comment by psywombats** [comment]
would also be nice to get these in a new chipset instead of just 'upper' and 'lower' ...or maybe the tiles are so small we can keep everything together?

**Comment by crumley467** [comment]
I think being added to these sets makes sense a least in that its still world 1?

**Comment by psywombats** [comment]
Yep makes sense.

Also some feedback about current tiles in the set: It's hard to use the wooden floors, tiled floors etc, because the limited color palette makes them extremely visually distracting. Not sure there's much that can be done about it, but it means they can't be used for things like large indoor dungeons, where I'm pretty much always going to be using the uniform gray chip. Can't remember if/how the FFL games solved this.

Ditto on dark walls. It's hard to have a dark wall with a dark wall topper, meaning the dark walls aren't seeing much use.

**Comment by crumley467** [comment]
Could you elaborate on visually distracting? And I'm pretty sure that FFL handled it by using the white/grey tiles and had very few building style floors. I only made those because i knew how from Blockbound. Also, how do you want the new chips uploaded? I know we talked a little, but I think maybe having a hero chipset would be a good idea

**Comment by psywombats** [comment]
I'd just add the new chips to the general chipsets. We probably won't be adding too many, really.

Re: visual stuff, check the oracle's palace with the cobblestone chip: http://i.imgur.com/bqUXWJ1.png
The black exterior helps a bit but it makes things look a bit busy. It's especially important on the outdoor maps (like the airseed shrine's first map) because there's the gravelly texture in the mix as well.

**Comment by psywombats** [comment]
Make sure that sparkling spacegate thing is in its own chipset though

**Comment by crumley467** [comment]
That's fair. You may have seen some of the other tile chips I added? The two right above the big pink space? They're a little less busy than the rest. A less busy wall I have is http://i.imgur.com/DFGSQBb.png that i have also just added to the chipset for a cleaner look for hero. Less grubby.
